server/commands/documentCreator.ts

```diff
@@ -101,14 +101,20 @@ export default async function documentCreator({
       fullWidth: templateDocument ? templateDocument.fullWidth : fullWidth,
       icon: templateDocument ? templateDocument.icon : icon,
       color: templateDocument ? templateDocument.color : color,
-      title: TextHelper.replaceTemplateVariables(
-        templateDocument ? templateDocument.title : title,
-        user
-      ),
-      text: TextHelper.replaceTemplateVariables(
-        templateDocument ? templateDocument.text : text,
-        user
-      ),
+      title:
+        title ??
+        (templateDocument
+          ? template
+            ? templateDocument.title
```

Comment:
IMO, it makes sense to retain template variables if we're creating a template from another template.

Reply:
clever, makes sense